awpy/parsers/utils.py

```diff
@@ -1,6 +1,7 @@
 """Module for parsing utils."""
 
 import polars as pl
+from loguru import logger
 
 
 def get_event_from_parsed_events(
```

Comment:
I would argue this should be `Optional[pl.DataFrame]` so that the type is large enough to have all possible outcomes (no list + empty list). But this might not be an issue in practice and would require some extra work elsewhere in the code. Plus, optionals are only good if their usage is enforced by the type system, so this is probably better.